app/routes/home.js

```diff
@@ -1,8 +1,12 @@
+const cache = require('../cache')
+
 module.exports = {
   method: 'GET',
   path: '/',
   options: {
-    handler: (request, h) => {
+    handler: async (request, h) => {
+      await cache.clear('progress', request.yar.id)
```

Comment:
I'm not sure it's a good idea to clear the cache when the go to the home screen as they will lose all their progress if they click the service banner.  But having said that until we've got management of in progress agreements restored, then it might not make sense to have it anywhere else other than here or on the button click to start a new agreement.